test/Microsoft.AspNet.Razor.Test/TagHelpers/TagHelperParseTreeRewriterTest.cs

```diff
@@ -8,16 +8,14 @@
 using Microsoft.AspNet.Razor.Generator;
 using Microsoft.AspNet.Razor.Parser;
 using Microsoft.AspNet.Razor.Parser.SyntaxTree;
-using Microsoft.AspNet.Razor.Parser.TagHelpers.Internal;
 using Microsoft.AspNet.Razor.TagHelpers;
 using Microsoft.AspNet.Razor.Test.Framework;
 using Microsoft.AspNet.Razor.Text;
-using Microsoft.AspNet.Razor.Tokenizer;
 using Xunit;
 
 namespace Microsoft.AspNet.Razor.Test.TagHelpers
 {
-    public class TagHelperParseTreeRewriterTest : CsHtmlMarkupParserTestBase
+    public class TagHelperParseTreeRewriterTest : TagHelperRewritingTestBase
```

Comment:
So we can re-use the base class bits. Why re-write it all? We already have a few examples of us doing this in Razor tests.